server/api/milestones/index.post.ts

```diff
@@ -15,7 +15,7 @@ export default defineEventHandler(async (event) => {
         equals: (filter.filters?.status as MilestoneStatus) || undefined,
       },
       journals: {
-        some: {
+        every: {
```

Comment:
Hope it's okay, otherwise milestones without journal id weren't included in response

Reply:
my bad